### PR TITLE
Add common test case

### PR DESCRIPTION
**Pull request by hokyungh**
### Description of changes:
Add common test case by @zhinang-amazon 
### Testing done:
Run unit test

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -20,7 +20,6 @@
 		0083A11423D8CFD40057027A /* RealtimeControllerFacade.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0083A11323D8CFD40057027A /* RealtimeControllerFacade.swift */; };
 		0083A11623D8CFFD0057027A /* DefaultRealtimeController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0083A11523D8CFFD0057027A /* DefaultRealtimeController.swift */; };
 		0095013423FF372E003260E3 /* VideoClientState.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0095013323FF372E003260E3 /* VideoClientState.swift */; };
-		00B45843246C745E0007D425 /* AmazonChimeSDKMedia.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5ADEAFA024293B2700801EA5 /* AmazonChimeSDKMedia.framework */; };
```
Comment (zhinang-amazon) [inline AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj:23]: Is this needed?

Comment (hokyungh) [reply AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj:23]: Not sure why this changed... I have added the AmazonChimeSDKMedia again

**Review by zhinang-amazon — APPROVED** [review]

File: AmazonChimeSDK/AmazonChimeSDKTests/common/CommonTestCase.swift [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@testable import AmazonChimeSDK
+import XCTest
+ 
+class CommonTestCase: XCTestCase {
```
Comment (alnlau) [inline AmazonChimeSDK/AmazonChimeSDKTests/common/CommonTestCase.swift:14]: I don't see any test cases here? If this is to provide mocks / setup for other tests to use, then perhaps something like `MockProvider`?

Also, why can't the other tests do the setup themselves?

**Comment by alnlau** [comment]
Let's squash the two commits as well.

File: AmazonChimeSDK/AmazonChimeSDKTests/common/CommonTestCase.swift (hunk 2 above)
Comment (zhinang-amazon) [reply AmazonChimeSDK/AmazonChimeSDKTests/common/CommonTestCase.swift:14]: Several test cases require the same mock and there is a lot of duplicate code without a place to host common mocks.

**Pull request closed without merging**